### ISSUE TITLE
Create test page for new icon evaluation

### ISSUE THREAD

**Issue by mceledonia** [patternfly/patternfly#4944]
This page should have a selection of icons displayed at each of our icon size variables. The purpose of this page is to evaluate the render quality of these icons at different display densities in the browser. Not intended to evaluate look+feel in context. 

CC @mcoker

**Comment by mceledonia** [comment]
Some interesting findings here. For context, I tested these on my windows dual monitor setup with a 27in 1080p and a 27in 1440p. Tested both 1080p and 1440p at native monitor resolutions, then dropped my 1440p monitor resolution through gpu settings to 720p.

1080p (27in, native)
![image](https://user-images.githubusercontent.com/30935938/177397559-cc692552-f11d-4ecc-9115-3e4661c7507f.png)

1440p (27in, native)
![image](https://user-images.githubusercontent.com/30935938/177397592-285287cf-9a67-4d26-9c49-4e3484892c09.png)

720p (27in, adjusted)
![image](https://user-images.githubusercontent.com/30935938/177397623-e891f275-99e8-44e0-b90e-4dafb10e42f8.png)

From what I can tell, these are not sitting on the pixel grid. Take note of horizontal or vertical lines and zoom in to see aliasing happening rather than getting a crisp
